proposals/improv-mac-os-vuln-detection.md

```diff
@@ -0,0 +1,117 @@
+# Improving vulnerability detection for MacOS
+
+[6001](https://github.com/fleetdm/fleet/issues/6001) identified some problems with our
+current approach to vulnerability detection on MacOS:
+
+- The version reported by software does not fit the standard format. For example, Zoom reports the version as 5.8.3 (2240).
+- The app name includes extra terms that don't appear in the title. For example, zoom.us is treated
```

Comment:
Am a bit out of the loop, could you define the following two?
- `app name`? (is this is the app name as defined by osquery?)
- `title`? (does this refer to the title in the `cpe` database?)